src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1496,6 +1546,10 @@ internal void Push(T item)
         /// <returns></returns>
         internal T Pop()
         {
+            if (this.First == null)
+            {
+                throw new InvalidOperationException(SessionStateStrings.BoundedStackIsEmpty);
+            }
             var item = this.First.Value;
```

Comment:
Please add new line after `}` (CodeFactor issue).

Reply:
I disagree, in order to improve readability I have put the code for `cd -` and cd +` into 2 blocks that are separated by a newline. This would make it worse, we should not be a slave of all rules, I question the usefulness of this rule. I agree that it can be useful to have a newline after a big block but not after a one-liner and not having braces can be dangerous as well.
I suggest that I could refactor the 2 blocks into a private method of their own instead, what do you think?

Reply:
@bergmeister I don't understand why you ask about private method. CodeFactor issue is to add new line after 1552 and before 1553. 
There are not many such issues in our code reported by CodeFactor. In other words, it is a style that we always follow.

Reply:
Sorry, I thought this comment was on the other place in the `SessionStateLocationAPIs` class.
The idea for that specific class was to put the code blocks into a private method to avoid making the main method even longer.
Nevertheless, I still find the rule odd for a brace that has only one line in it but I will have to comply with whatever is preferred at the end anyway.

Reply:
No one happy in this situation (style issues) but at least this tool always gives the same result as opposed to human work. 
Now the CodeFactor settings are such that we get a minimum of requests for each PR. I hope in a few months we will be able to escape most annoying style issues.